src/screens/Signup/StepInfo/index.tsx

```diff
@@ -27,10 +27,18 @@ function sanitizeDate(date: Date): Date {
   return date
 }
 
-export function StepInfo() {
+export function StepInfo({
+  isLoadingStarterPack,
+}: {
+  isLoadingStarterPack: boolean
+}) {
   const {_} = useLingui()
   const {state, dispatch} = useSignupContext()
 
+  useEffect(() => {
+    LayoutAnimation.configureNext(LayoutAnimation.Presets.easeInEaseOut)
```

Comment:
This is kind of a race condition, effects have no well-defined lifetime in RN right now so this call may align with some different paint. Can we move this closer to the setstate we actually want to animate?

Reply:
going to leave for now and see per conversation.